service/rpc/servicer.go

```diff
@@ -419,6 +419,98 @@ func (s *Servicer) BlockTransaction(ctx context.Context, request *types.BlockTra
 	}, nil
 }
 
+type CallMethod string
+
+const CeloCall CallMethod = "celo_call"
+const CeloGetLogs CallMethod = "celo_getLogs"
+
+func (cm CallMethod) String() string { return string(cm) }
+
+func AllCallMethods() []string {
+	return []string{CeloCall.String(), CeloGetLogs.String()}
+}
+
+func (s *Servicer) Call(ctx context.Context, request *types.CallRequest) (*types.CallResponse, *types.Error) {
+
+	switch request.Method {
+	case CeloCall.String():
+		var callArgs *airgap.CallArgs
+		if err := airgap.UnmarshallFromMap(request.Parameters, callArgs); err != nil {
+			return nil, LogErrValidation(err)
+		}
+
+		// TODO: allow for block number in parameters; nil == latest
+		data, err := s.airgap.CallData(ctx, callArgs, nil)
+		if err != nil {
+			return nil, LogErrCeloClient(request.Method, err)
+		}
+
+		return &types.CallResponse{
+			Result: map[string]interface{}{
+				"raw": data,
+			},
+			Idempotent: false,
+		}, nil
+	case CeloGetLogs.String():
```

Comment:
@yorhodes any progress on implementing the logs call? happy to help out if needed